niworkflows/interfaces/reportlets/base.py

```diff
@@ -52,6 +52,7 @@ class RegistrationRC(reporting.ReportCapableInterface):
     _fixed_image_label = "fixed"
     _moving_image_label = "moving"
     _contour = None
+    _rotate_to_canonical = False
```

Comment:
```suggestion
    _dismiss_affine = False
```

I think I would be more explicit about not considering the affine.